### DISCUSSION TITLE
Change author profile width

### DISCUSSION THREAD

**Discussion by jadeidev** [mmistakes/minimal-mistakes#2862] [Q&A]
## Environment

- Minimal Mistakes version: 4.22
- Ruby gem or remote theme version: remote

I tried to tinker around with the scss variables `$right-sidebar-width-narrow` and `$right-sidebar-width` in an attempt to change the author profile width, but obviously that would affect the TOC. I am wondering if there is a way to just change the author profile width. In particular i want to make it narrower for large and x-large without affecting the TOC

**Comment by gstotts** [comment]
Are you using the wide class or are you wishing to make even further adjustments to the width than that?
